css/index.css

```diff
@@ -2,3 +2,221 @@
 This imports the `utility.css` file */
 @import url('./reset.css');
 @import url('./utility.css');
+@import url('https://fonts.googleapis.com/css2?family=Spartan:wght@700&display=swap');
+
+.theme1 {
+  --bg-theme: hsl(222deg 26% 31%);
+  --bg-toggle-keypad: hsl(223deg 31% 20%);
+  --bg-screen: hsl(224deg 36% 15%);
+  --bg-del-reset-key: hsl(225deg 21% 49%);
+  --key-shadow-del-reset: hsl(224deg 28% 35%);
+  --bg-keys: hsl(30deg 25% 89%);
+  --key-shadow-keys: hsl(28deg 16% 65%);
+  --bg-result-toggle-key: hsl(6deg 63% 50%);
+  --key-shadow-result: hsl(6deg 70% 34%);
+  --text-color: hsl(30deg 25% 89%);
+  --text-del-result-color: hsl(0deg 0% 100%);
+  --text-result-color: hsl(221deg 14% 31%);
+  --equals-color: hsl(0deg 0% 100%);
+  --dot-position: 8px;
+  --del-reset-hover-color: #a2b3e1;
+  --key-hover: #fff;
+  --reset-dot-hover: #f96c5b;
```

Comment:
Let's change this to `hsl` to make the color consistent

Reply:
Done